### ISSUE TITLE
Monster knowledge issues (more!)

### ISSUE THREAD

**Issue by angband-import** [angband/angband#785]
**Reported by takkaria on 12 Jul 2007 11:29 UTC**
Harpies are in the hydras group, as well as in the hybrids group.

[http://angband.oook.cz/rgra.php?showpost=103963]

**Comment by angband-import** [comment]
**Comment by takkaria on 14 Jul 2007 09:19 UTC**
```
Index: src/cmd4.c
===================================================================
--- src/cmd4.c  (revision 337)
+++ src/cmd4.c  (working copy)
\@\@ -148,9 +148,9 \@\@
        { "g",          "Golems" },
        { "H",          "Harpies/Hybrids" },
        { "h",          "Hominids (Elves, Dwarves)" },
-       { "H",          "Hydras" },
+       { "M",          "Hydras" },
        { "i",          "Icky Things" },
-       { "FI",         "Insects" },
+       { "lFI",                "Insects" },
        { "j",          "Jellies" },
        { "K",          "Killer Beetles" },
        { "k",          "Kobolds" },
```

**Comment by angband-import** [comment]
**Comment by ajps on 14 Jul 2007 12:50 UTC**
Ta for the patch - fixed in [339] merged in [5dfe994] (SVN r399).
